packages/components/package.json

```diff
@@ -0,0 +1,76 @@
+{
+  "name": "@squiggle/components",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@squiggle/lang": "^0.1.9",
+    "@testing-library/jest-dom": "^5.16.2",
+    "@testing-library/react": "^12.1.2",
+    "@testing-library/user-event": "^13.5.0",
+    "@types/jest": "^27.4.0",
+    "@types/lodash": "^4.14.178",
+    "@types/node": "^17.0.16",
+    "@types/react": "^17.0.39",
+    "@types/react-dom": "^17.0.11",
+    "cross-env": "^7.0.3",
+    "lodash": "^4.17.21",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
+    "react-scripts": "5.0.0",
+    "react-vega": "^7.4.4",
+    "tsconfig-paths-webpack-plugin": "^3.5.2",
+    "typescript": "^4.5.5",
+    "vega-embed": "^6.20.6",
+    "web-vitals": "^2.1.4",
+    "webpack-cli": "^4.9.2"
+  },
+  "scripts": {
+    "start": "react-scripts start",
```

Comment:
My impression is that we don't actually use the app part of this? Just the storybook aspect? It might be possible to remove a lot of the regular app aspects then. (Not sure what we'd use them for)